cmd/server.go

```diff
@@ -1062,12 +1106,32 @@ var serverCreate = &cobra.Command{
 			opt.EnablePrivateNetwork = true
 		}
 
-		server, err := client.Server.Create(context.TODO(), region, plan, osID, opt)
+		server, err := client.Server.Create(context.TODO(), region, plan, osOpt, opt)
 
 		if err != nil {
-			fmt.Printf("error setting creating instance : %v", err)
+			fmt.Printf("error creating instance : %v", err)
```

Comment:
```suggestion
			fmt.Printf("error creating instance : %v\n", err)
```
Printf doesn't print the trailing newline itself.